src/diffusers/loaders/lora_pipeline.py

```diff
@@ -2113,8 +2195,83 @@ def unfuse_lora(self, components: List[str] = ["transformer", "text_encoder"], *
         Args:
             components (`List[str]`): List of LoRA-injectable components to unfuse LoRA from.
         """
+        transformer = getattr(self, self.transformer_name) if not hasattr(self, "transformer") else self.transformer
+        transformer.load_state_dict(self._transformer_norm_layers)
```

Comment:
Should this be called with `strict=False`? 

Also great way to deal with this!